libs/libc/string/lib_strcoll.c

```diff
@@ -56,4 +56,9 @@ int strcoll(const char *a, const char *b)
 {
   return strcmp(a, b);
 }
+
+int strcoll_l(const char *a, const char *b, locale_t l)
```

Comment:
```suggestion
int strcoll_l(FAR const char *a, FAR const char *b, locale_t l)
```
Missing `FAR` qualifier.

Reply:
Done.